### PR TITLE
add new command: algob sign-lsig

### PR DESCRIPTION
**Pull request by ratik21**
added new command `algob sign-lsig --account <acc> --file <input> --out <out-file>` to sign logic signature fetched from file

### CODE REVIEW

File: packages/algob/src/lib/lsig.ts [hunk 1]
```diff
@@ -27,8 +28,30 @@ export async function getLsig (
...
 }
+
+/**
+ * Appends signature (using signer's sk) to multi-signed logic signature. If multisig is not found
+ * then new multisig is created
+ * eg. appending own signature to a signed lsig (received from multisignature account address network)
+ * @param lsig Logic Sig object
+ * @param signer: Signer Account which will sign the smart contract
+ * @param mparams: passed when signing a new multisig
+ * @returns multi signed logic signature (with appended signature from signer's sk)
```
Comment (robert-zaremba) [inline packages/algob/src/lib/lsig.ts:44]:
```suggestion
 * @returns multi signed logic signature (with appended signature using signer's sk)
```

**Review by robert-zaremba — COMMENTED** [review]
This is great. Test is missing - would you be able to add a test? I would like to include this functionality in the Release.

**Review by robert-zaremba — APPROVED** [review]
pre-approving - but we still need a test.

**Comment by ratik21** [comment]
Added tests.

**Pull request merged**